src/action/channel.js

```diff
@@ -87,6 +85,14 @@ class ChannelAction {
     ]);
   }
 
+  /**
+   * Poll the channels in the background since there is no streaming grpc api
+   * @return {Promise<undefined>}
+   */
+  async pollChannels() {
+    await poll(() => this.update());
```

Comment:
I think it would be a good idea to increase the polling time out (at least for channels) to 5 or 10 seconds. If this helps people's overheating problems, we know that the polling was the cause.

Reply:
Fixed this in b4c3e23